src/covid_model_seiir_pipeline/pipeline/forecasting/task/beta_forecast.py

```diff
@@ -42,10 +42,10 @@ def run_beta_forecast(forecast_version: str, scenario: str, draw_id: int):
     # Covariates and coefficients, and scaling parameters are
     # used to compute beta hat in the future.
     covariates = data_interface.load_covariates(scenario_spec)
-    coefficients = data_interface.load_regression_coefficients(draw_id)
+    coefficients = data_interface.load_regression_coefficients(draw_id).reset_index()
     forecast_end_date = covariates.date.max()
     # Rescaling parameters for the beta forecast.
-    beta_scales = data_interface.load_beta_scales(scenario=scenario, draw_id=draw_id)
+    beta_scales = data_interface.load_beta_scales(scenario=scenario, draw_id=draw_id).reset_index()
```

Comment:
These two changes suggest to me that `load_foo` methods should all reset the index before returning their result. thoughts?

Reply:
I'm leaving it up at this level.  I introduce an index abstraction in a later PR in the forecast stage.  I want the noise here at this level to annoy me into backporting it into regression soon.